### PR TITLE
[Proposal] Rename prospectors to inputs

### PR DESCRIPTION
**Pull request by ruflin**
This change would require to update all docs.

### CODE REVIEW

File: filebeat/beater/filebeat.go [hunk 1]
```diff
@@ -40,6 +40,11 @@ func (fb *Filebeat) Config(b *beat.Beat) error {
 	// Check if optional config_dir is set to fetch additional prospector config files
 	fb.config.FetchConfigs()
 
+	if len(fb.config.Filebeat.Prospectors) > 0 {
+		fb.config.Filebeat.Inputs = fb.config.Filebeat.Prospectors
```
Comment (urso) [inline filebeat/beater/filebeat.go]: what if I have both, `inputs` and `prospectors`?

Comment (ruflin) [inline filebeat/beater/filebeat.go]: Not allowed. In case we decide to proceed, I will add a check for that. In case both are set, and error is returned.

**Comment by tsg** [comment]
Nice, changeset is small enough. But it's still a big change as the new terminology changes a bit the way we think about the way this works. So I'd say let's all sleep over it before pressing merge :-).

**Comment by ruflin** [comment]
Closing this PR as there are too many filebeat changes in the pipeline before beta1 I want to focus on (#1600). I don't think we should rush any changes here as it is not just about renaming. It also affects the docs and knowledge of existing users.

**Pull request closed without merging**